### PR TITLE
create simily00.md

### PR DESCRIPTION
**Pull request by simily00**
Fixes #

### Description
Simily's first pull.

### RawGit preview link

https://rawgit.com/simily00/SimilyHill.github.io/my-new-branch-name/#!pages/vi/profiles/simily00.md

### CODE REVIEW

**Comment by yuviii** [comment]
@simily00 I recommend u to use your own picture instead of city.

**Comment by lmmrssa** [comment]
@simily00 link you used for image is referenced to your local machine could you have public url

File: pages/vi/profiles/simily00.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Howdy! my name is Simily Hill.
+## I am from sunny California (PTD), the Bay Area to be exact. 
+ ![Alt Text](http://res.cloudinary.com/d1991/image/upload/c_scale,h_176,w_263/v1536641344/meinbraids_ep77mu.jpg)
```
Comment (simily00) [inline pages/vi/profiles/simily00.md]: Image change-update

File: pages/vi/profiles/simily00.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+------------ | ------------- | -----------
+ I am an Audiophile  :musical_note: |  Spiders :spider: | Junior at SF State studing Computer Science  :mortar_board:
+ Boba tea enthusiast  :tea:|  Ants  :ant:|  I am an amazing driver  :checkered_flag:
+
+
+
+
+**Check out some fun stuff I did here-->** [fun stuff click me!](http://www.similythesitebuilder.com).
+
+ - [x] If Simily seems awesome check here
```
Comment (simily00) [inline pages/vi/profiles/simily00.md]: Added additional markdown example

File: pages/vi/profiles/simily00.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Howdy! My name is Simily Hill.
```
Comment (simily00) [inline pages/vi/profiles/simily00.md]: Corrected punctuation

File: pages/vi/profiles/simily00.md (hunk 2 above)
Comment (simily00) [inline pages/vi/profiles/simily00.md]: Added additional markdown example

File: pages/vi/profiles/simily00.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+## I am from sunny California (PTD), the Bay Area to be exact. 
+ ![Alt Text](http://res.cloudinary.com/d1991/image/upload/c_scale,h_176,w_263/v1536641344/meinbraids_ep77mu.jpg)
+
+_OS X El Capitan_ *(version 10.11.6)*
+### Some things about me:
+
+Interest | Fears :warning: | Facts
```
Comment (simily00) [inline pages/vi/profiles/simily00.md]: Inserted markdown table

File: pages/vi/profiles/simily00.md (hunk 2 above)
Comment (dogi) [inline pages/vi/profiles/simily00.md]: - [x] awesomest

**Pull request merged**